pytorch_lightning/accelerators/accelerator.py

```diff
@@ -355,12 +355,8 @@ def clip_gradients(
             model=self.model,
         )
 
-    def on_train_epoch_end(self, outputs: EPOCH_OUTPUT) -> None:
-        """Hook to do something on the end of an training epoch
-
-        Args:
-            outputs: the outputs of the training steps
-        """
+    def on_train_epoch_end(self) -> None:
+        """Hook to do something on the end of an training epoch."""
```

Comment:
```suggestion
        """ Hook to do something on the end of an training epoch. """
```

Reply:
i kept it like this for consistency with the other hooks. maybe we can clean this up in a separate PR to minimize the changes here?

Reply:
Yeah, works for me